src/abstractdataframe/abstractdataframe.jl

```diff
@@ -365,39 +369,58 @@ If the column does not allow missing values, `nothing` is returned.
 Consequently, `nmissing = 0` indicates that the column allows
 missing values, but does not currently contain any.
 
+Custom functions perform call `skipmissing` on columns of eltype `Union{T, Missing}`, 
```

Comment:
I suggest something like this:
```julia
If custom functions are provided, they are called repeatedly with the vector corresponding
to each column as the only argument. For columns allowing for missing values,
the vector is wrapped in a call to [`skipmissing`](@ref): custom functions must therefore
support such objects (and not only vectors), and cannot access missing values.
```

The last sentence isn't true right now since we allocate a vector without missing values, but we could avoid doing that in the future when neither the median nor quantiles need to be computed. That would be much faster.

Reply:
fixed.